patches/export_pdf.patch

```diff
@@ -0,0 +1,131 @@
+diff --git a/pdf/pdf.cc b/pdf/pdf.cc
+index 7aba412..246cd81 100644
+--- a/pdf/pdf.cc
++++ b/pdf/pdf.cc
+@@ -81,21 +81,26 @@ const void* PPP_GetInterface(const char* interface_name) {
+   return pp::Module::Get()->GetPluginInterface(interface_name);
+ }
+ 
++}  // namespace chrome_pdf
++
++
++extern "C" {
++
+ #if defined(OS_WIN)
+-bool RenderPDFPageToDC(const void* pdf_buffer,
+-                                 int buffer_size,
+-                                 int page_number,
+-                                 HDC dc,
+-                                 int dpi,
+-                                 int bounds_origin_x,
+-                                 int bounds_origin_y,
+-                                 int bounds_width,
+-                                 int bounds_height,
+-                                 bool fit_to_bounds,
+-                                 bool stretch_to_bounds,
+-                                 bool keep_aspect_ratio,
+-                                 bool center_in_bounds,
+-                                 bool autorotate) {
++PDF_EXPORT bool RenderPDFPageToDC(const void* pdf_buffer,
```

Comment:
> I think PDF_EXPORT is more suitable to be put in the pdf.h?

The reason for it is that to make the patch minimum. I have tried it before, it will lead to more change in `pdf.h`.